crates/kitsune_p2p/types/src/config.rs

```diff
@@ -166,10 +202,13 @@ pub mod tuning_params_struct {
         ///   a good reason, such as being on a bandwidth constrained
         ///   mobile device!
         /// - "full" - Indicates that you commit to serve and hold all
-        ///   all data from all agents, and be a potential target for all
+        ///   data from all agents, and be a potential target for all
         ///   get requests. This could be a significant investment of
         ///   bandwidth. Don't take this responsibility lightly.
+        #[cfg(feature = "unstable-sharding")]
         gossip_arc_clamping: String = "none".to_string(),
+        #[cfg_not(feature = "unstable-sharding")]
```

Comment:
Is `cfg_not` a thing? Didn't know that.

Reply:
It is not, it's text-matched above

If you try to make a macro match either:
- #[cfg(not(...
- #[cfg(...

Then the second `...` would be a valid match for `not(...` and you end up with an ambiguous parser. So I invented new syntax that expands to `cfg(not(`. It made me rather unhappy :)

Reply:
Exactly, in there :D

Reply:
I'm thinking we'll have to re-work that macro/type to support app-supplied Kitsune params anyway so I'm not too concerned about the complexity there